packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -61,7 +61,7 @@ export const DetailView: FC = () => {
   return draft ? (
     <TableProvider createStore={createTableStore}>
       <AppBarButtons
-        isDisabled={isInvoiceEditable(draft)}
+        isDisabled={!isInvoiceEditable(draft)}
```

Comment:
it was proving to be a pain when testing the add items!

Reply:
oops!